src/services/login.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import Utils from './utils';
+
+export async function login(email, password) {
+    return await axios
+    .post(`${Utils.getBaseUrl()}/login`, {
+    email: email,
+    password: password,
```

Comment:
```suggestion
    email,
    password,
```
É apenas questão estética, não é preciso alterar, se quiser pode seguir em frente mesmo com essa sugestão.